.circleci/config.yml

```diff
@@ -1304,15 +1304,6 @@ workflows:
            - CI_MEASURE
   gcp-ehp:
     jobs:
-      - build-and-push-image:
```

Comment:
Je pense que dans build-and-push-image on a plus besoin du app version : il ne reste plus que les cas où cette app_version vaut CIRCLE_SHA1, et est donc déjà couvert par le job lui même

Reply:
Bien vu !